bootstrapvz/providers/ec2/__init__.py

```diff
@@ -89,6 +89,7 @@ def resolve_tasks(taskset, manifest):
 
 	if manifest.volume['partitions']['type'] != 'none':
 		taskset.add(initd.AdjustExpandRootScript)
+		taskset.add(initd.AdjustGrowpartWorkaround)
```

Comment:
Since this is now its own task, you should do the `>=jessie` check here. Otherwise the task is a no-op for earlier distributions.